allennlp_models/modelcards/tagging-elmo-crf-tagger.json

```diff
@@ -38,7 +38,8 @@
         "dataset": {
             "name": "CoNLL-2003 NER dataset",
             "url": "https://www.clips.uantwerpen.be/conll2003/ner/",
-            "notes": "The NER model was evaluated on the CoNLL-2003 NER dataset. Unfortunately we cannot release this data due to licensing restrictions."
+            "notes": "The NER model was evaluated on the CoNLL-2003 NER dataset. Unfortunately we cannot release this data due to licensing restrictions.",
+            "processed_url": "path/to/dataset"
```

Comment:
It's better to leave this empty as we don't have a path/to/dataset. The field `processed_url` is interpreted as a link i the UI.

Reply:
On second thoughts, since the training command is constructed using the link, perhaps it's ok to have a placeholder.

Reply:
That's why I put it there. Otherwise it doesn't print the command at all.